concordia/migrations/0024_auto_20190410_1213.py

```diff
@@ -0,0 +1,54 @@
+# Generated by Django 2.2 on 2019-04-10 16:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("concordia", "0023_auto_20190130_1555")]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name="sitereport", options={"ordering": ("created_on",)}
+        ),
+        migrations.AddField(
+            model_name="assettranscriptionreservation",
+            name="session_id",
+            field=models.CharField(blank=True, max_length=50, null=True),
+        ),
+        migrations.AlterField(
```

Comment:
I'm not sure why this migration came out like this - I will look into it.

Reply:
I guess there were some changes made to the models, or changes made in the underlying Django assumptions, that caused this migration to have non-reserve-id related changes in it. I loaded a fresh database and still got the same result.

Reply:
I created https://github.com/LibraryOfCongress/concordia/pull/904 to handle that separately so this PR can stay clean